### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by subintp**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Is it a breaking change?

How do we intend to rollback or handle backward compatibility in case something breaks?
Can this change be tested effectively via canary?

## Screenshots:

Include a screenshot of the changes, if applicable.

# Checklist Exception

- [ ] My change doesn't require the below checklist to be updated.
Please add a description on why it is not applicable.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added unit tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added integration tests for the new feature (if applicable)
- [ ] I have manually tested my code to the best of my abilities.

### CODE REVIEW

File: tests/functional/main_test.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func setup() {
 	}
 
 	hname := os.Getenv("HOSTNAME")
-	if env == "" {
+	if env == "" || env == "dev" {
```
Comment (vnktram) [inline tests/functional/main_test.go:61]: @subintp Is this change intentional?

Comment (subintp) [reply tests/functional/main_test.go:61]: yes. Ref: https://github.com/razorpay/metro/commit/6f904b6c82bbe57a9854d50f4890dc0a28597f15#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R88

**Review by sakshi-goyal-razorpay — APPROVED** [review]
